src/dashboards/hardware-sentry/Hardware Sentry - Site.json

```diff
@@ -3591,21 +3685,25 @@
               "Time 1": true,
               "Time 2": true,
               "Time 3": true,
+              "Time 6": true,
```

Comment:
We should probably modify the PromQL in this panel so that only the labels that we need is provided in the result, to avoid having to deal with "id 1", "id 2", "id 3", "id 4", "id 5" and "id 6" for example.